nuxt.config.ts

```diff
@@ -31,12 +31,9 @@ const meta = [
     name: "viewport",
     content: "width=device-width,initial-scale=1",
   },
-  // By default, tell all robots not to index pages. Will be overridden in the
+  // By default, tell all robots not to index pages. Will be overwritten in the
```

Comment:
Interesting.

Reply:
I have no idea where this came from, I'll remove it.